xchange-therock/src/main/java/org/knowm/xchange/therock/dto/marketdata/TheRockTicker.java

```diff
@@ -3,21 +3,71 @@
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import java.io.Serializable;
 import java.math.BigDecimal;
 import java.util.Date;
 import org.knowm.xchange.currency.CurrencyPair;
-import org.knowm.xchange.utils.jackson.CurrencyPairDeserializer;
+import org.knowm.xchange.therock.utils.jackson.TheRockTickerDeserializer;
 
 /** @author Matija Mazi */
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
-public class TheRockTicker {
+@JsonDeserialize(using = TheRockTickerDeserializer.class)
+public class TheRockTicker implements Serializable {
+  /** */
+  private static final long serialVersionUID = -4662345648477360478L;
 
-  @JsonDeserialize(using = CurrencyPairDeserializer.class)
   private CurrencyPair fundId;
```

Comment:
I think we could also just change this to 

```
private String fundId;
```

mapping this to an CurrencyPair here is to early

an adapter should do this - even if the mapping is trivial

Reply:
shall we just leave this unchanged, it seems to work with the custom instrument deserializer.